easydiffraction/calculators/cryspy.py

```diff
@@ -218,8 +220,25 @@ def assignCell_toCrystal(self, cell_name: str, crystal_name: str):
         crystal.cell = cell
 
     def createSpaceGroup(
-        self, key: str = "spacegroup", name_hm_alt: str = "P 1"
+        self, key: str = "spacegroup", name_hm_alt: str = "", it_code: Optional[str] = ""
     ) -> str:
+        if not name_hm_alt and not self.hm_symbol:
```

Comment:
The meaning of `self.hm_symbol` is the same as `name_hm_alt`, right? Wouldn't it then be more readable to rename `self.hm_symbol` to `self.name_hm_alt`?